scylla-cdc-base/src/main/java/com/scylladb/cdc/model/worker/ChangeSchema.java

```diff
@@ -1,289 +1,306 @@
-package com.scylladb.cdc.model.worker;
```

Comment:
Why is this diff so large? (It first deletes the entire file then adds it back combined with your changes)

Reply:
Glö. I am gonna go ahead and say 'git rebase' mongo f**kup. I'll try to fix the patch to behave proper.

Reply:
Should be fixed now.